### DISCUSSION TITLE
Is .ts available

### DISCUSSION THREAD

**Discussion by tompointexe** [bluenviron/mediamtx#812] [Questions]
Hey, i'm looking to input the generated hls stream to vrchat on quest, and i'm seeing sucess using a .ts format, any way to enable the .ts extention at the end of my url for vlc / vrchat
i've tryed http://server:8888/streamkey/index.ts with no sucess :(

**Comment by tompointexe** [comment]
I think i'm dumb and i just need to add https to the webserver part

**Reply by aler9** [reply]
The server doesn't provide streams in MPEG-TS format, the server provides streams in HLS format, you can access them with VLC and this URL:

```
http://localhost:8888/mystream/index.m3u8
```

as explained in the README:

https://github.com/aler9/rtsp-simple-server#hls-general-usage

[a comment by an automated account is not shown]
